src/unity/lib/annotation/image_classification.cpp

```diff
@@ -238,7 +238,7 @@ void ImageClassification::_addAnnotationToSFrame(size_t index, int label) {
 
 void ImageClassification::cast_annotations() {
   size_t annotation_column_index = m_data->column_index(m_annotation_column);
-  if(m_data->dtype().at(annotation_column_index) == flex_type_enum::INTEGER) {
+  if (m_data->dtype().at(annotation_column_index) == flex_type_enum::INTEGER) {
```

Comment:
I know this is reformatted old code, but this should have been m_data->column_type(annotation_column_index) in the first place.  Or just m_data->column_type(m_annotation_column).

Reply:
I don't think `unity_sframe` has this method on it. Unless you're suggesting I use a different data representation.